tests/testing/make_sdists.py

```diff
@@ -6,12 +6,13 @@
 def random_string():
     """return a short suffix that shouldn't collide with any subsequent calls"""
     import os
-    import base64
+    import string
+    import random
 
-    return '.'.join((
-        str(os.getpid()),
-        base64.urlsafe_b64encode(os.urandom(3)),
-    ))
+    return '{}.{}'.format(
+        os.getpid(),
+        ''.join(random.choice(string.ascii_lowercase) for x in range(8)),
```

Comment:
latin1 is much wider than needed
b64 is US-ASCII

> what's the point of using base64 to get a random string?
So that random bytes are readable, representable in a file name.